01-wasmcloud_basics/wash_intro/start.md

```diff
@@ -0,0 +1,36 @@
+## Starting wash
+
+`wash` is a command line interface (CLI) that also supports a fully interactive REPL mode. In order to start the `wash` REPL, we run `wash up`
+
+> Note: This is being run for you after the environment has fully bootstraped. This can take as long as 2 minutes.
+
+```bash
+❯ wash up --help
+wash-up 0.3.0
+Launch wasmcloud REPL environment
+
+USAGE:
+    wash up [OPTIONS]
+
+FLAGS:
+        --help    Prints help information
+
+OPTIONS:
+    -l, --log-level <log-level>    Log level verbosity, valid values are `error`, `warn`, `info`, `debug`, and `trace`
+                                   [default: debug]
+    -h, --host <rpc-host>          Host for lattice connections, defaults to 0.0.0.0 [env: WASH_RPC_HOST=]  [default:
+                                   0.0.0.0]
+    -p, --port <rpc-port>          Port for lattice connections, defaults to 4222 [env: WASH_RPC_PORT=]  [default:
+                                   4222]
+```
```

Comment:
I would just remove this section, it doesn't display quite right in the katacoda pane, and you detail the REPL below. I would suggest adding a little snippet here though, asking the user to make the window as big as possible to ensure things display well.